staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/BUILD

```diff
@@ -2,7 +2,10 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 go_library(
     name = "go_default_library",
-    srcs = ["fieldmanager.go"],
+    srcs = [
+        "fieldmanager.go",
+        "lazy_fieldmanager.go",
```

Comment:
I don't know if I'm a big fan of the name. "Lazy" doesn't really say why it's being lazy. What about: `TrackAppliedFieldManager`, that's pretty bad, but you get the idea.

Reply:
I'll think about it

Reply:
Maybe SkipNonAppliedFieldManager?